### DISCUSSION TITLE
"openIsar" is not a function

### DISCUSSION THREAD

**Discussion by emmanuelagarry** [isar/isar#72] [Help]
The function "openIsar" does not exist after importing Isar in my projects.
 ![image](https://user-images.githubusercontent.com/26308935/114663361-73978900-9cfa-11eb-8bfd-79b020ac1752.png)

**Comment by simc** [comment]
@emmanuelagarry You need to run the `build_runner` and import `isar.g.dart`

**Comment by bare-metal-gpu** [comment]
what should it generate for? Like what annotation does it search for?

**Comment by leonfuss** [comment]
ou can find all of this perfectly documented on [isar.dev]()

**Comment by bare-metal-gpu** [comment]
SOLVED: yeah I just discovered the docs website - missed the @Id annotation inside the collection. Then it generated
